examples/neon-snake/src/logic/logicConfig.ts

```diff
@@ -1,10 +1,12 @@
 export const countdownDurationSeconds = 5
 export const endOfRoundDurationSeconds = 4
-export const maxScoreBeforeGameOver = 10
+export const maxScoreBeforeGameOver = 5
 export const boardSize = { width: 600, height: 900 }
 export const movePixelsPerTick = 2.5
-export const turnDegreesPerTick = 3
+export const turnDegreesPerTick = 5
 export const gapChancePerTick = 0.01
 export const gapLengthTicks = 20
 export const minTicksToNextGap = 30
 export const colors = ["#BCFE00", "#10D4FF", "#FF32D2", "#FF9C27"]
+
+export const ALLOWED_COLLISION_POINTS = 3
```

Comment:
```suggestion
export const allowedCollisionPoints = 3
```

FFTI: To keep in same style as remaining code